controllers/depresolver/depresolver_config.go

```diff
@@ -76,7 +76,7 @@ func (dr *DependencyResolver) ResolveOperatorConfig() (*Config, error) {
 		dr.config.Override.FakeDNSEnabled = env.GetEnvAsBoolOrFallback(OverrideWithFakeDNSKey, false)
 		dr.config.Override.FakeInfobloxEnabled = env.GetEnvAsBoolOrFallback(OverrideFakeInfobloxKey, false)
 		dr.config.Log.Level, _ = zerolog.ParseLevel(strings.ToLower(env.GetEnvAsStringOrFallback(LogLevelKey, zerolog.InfoLevel.String())))
-		dr.config.Log.Format = parseLogOutputFormat(strings.ToLower(env.GetEnvAsStringOrFallback(LogFormatKey, simple)))
+		dr.config.Log.Format = parseLogOutputFormat(strings.ToLower(env.GetEnvAsStringOrFallback(LogFormatKey, JSON.String())))
```

Comment:
Just thinking: `JSON`, `depresolver.JSON`, `Simple`,`depresolver.Simple` looks a bit unusual in the code, hard to understand the purpose without knowing the context, probably need to add `Format` post-fix to these "enum"-like constants, e.g. `JSONFormat`, `SimpleFormat` etc., the way how zerolog does it (`zerolog.InfoLevel`)

Reply:
@somaritane amended